v2/cmd/car/split.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	carv2 "github.com/ipld/go-car/v2"
+	"github.com/urfave/cli/v2"
+)
+
+// SplitCar is a command to output the index part of a car.
+func SplitCar(c *cli.Context) error {
+	r, err := carv2.OpenReader(c.Args().Get(0))
+	if err != nil {
+		return err
+	}
+	defer r.Close()
+
+	if !r.Header.HasIndex() {
+		return fmt.Errorf("no index present")
+	}
+	_, err = io.Copy(os.Stdout, r.IndexReader())
```

Comment:
consider making this a second argument on disk, because printing to a terminal will almost never be a good idea.